reports/issue-count-cycle-time.ts

```diff
@@ -0,0 +1,124 @@
+import {ProjectData, ProjectIssue, IssueCardEventProject} from '../interfaces';
+import * as rptLib from '../project-reports-lib';
+const tablemark = require('tablemark')
+import * as os from 'os';
+
+const reportType = 'project';
+export {reportType};
+
+export type CycleTimeData = { [key: string]: CycleTimeStageData }
+interface CycleTimeStageData {
+  title: string,
+  cycletime: number,
+  limit: number,
+  count: number,
+  flag: boolean
+}
+
+interface CycleTimeRow {
+    worktype: string,
+    labels: string,
+    count: number,
+    cycleTimeInDays: string,
+    limit: number
+    flag: string
+}
+
+export interface IssueCardCycleTime extends ProjectIssue {
+  cycletime: number
+}
+
+export function getDefaultConfiguration(): any {
+    return <any>{
+      "report-on": ["bug", "p2", "service" ],
```

Comment:
rename to `report-on-labels` to be consistent.  I changed to that in other reports after this PR was created.